exercises/practice/quadratic-roots/quadratic_roots_test.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from quadratic_roots import roots
+
+
```

Comment:
Remove extra white space

Reply:
[PEP 8](https://www.python.org/dev/peps/pep-0008/#introduction) recommends 2 blank lines between function definitions, so I was sticking to that. If there's a different style that we're using I can switch it.

Reply:
At the 6 companies that I've co-oped at and worked at full time it's always a standard to only leave 1 space. Not sure if that's an industry standard or just an industry preference